tests/target/issue-4689/two.rs

```diff
@@ -23,7 +23,8 @@ pub trait PrettyPrinter<'tcx>:
         Type = Self,
         DynExistential = Self,
         Const = Self,
-    > + fmt::Write
+    >
+    + fmt::Write
```

Comment:
Taking a look at the [style guide description for traits](https://doc.rust-lang.org/nightly/style-guide/items.html#traits)

**emphasis mine**:

> Prefer not to line-break in the bounds if possible (consider using a where clause). Prefer to break between bounds than to break any individual bound. **_If you must break the bounds, put each bound (including the first) on its own block-indented line, break before the +_** and put the opening brace on its own line:

I think these changes align more closely with the language in the guide than the formatting that we had before, but please let me know if I'm misinterpreting the guide.